examples/sftp-service-send-file/sftp_service_send_file.metatags

```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote SFTP server and using the `ftp:Caller` to interact with the SFTP server.
+description: This example demonstrates receiving file/directory changes that occur in a remote SFTP server and using an ftp:Caller to interact with the SFTP server.
```

Comment:
Here also the description can be improved

Reply:
Fixed